module/behaviour/strategy/KeyboardWalk/src/KeyboardWalk.hpp

```diff
@@ -87,6 +86,10 @@ namespace module::behaviour::strategy {
         /// @brief Called by the powerplant to build and setup the KeyboardWalk reactor.
         explicit KeyboardWalk(std::unique_ptr<NUClear::Environment> environment);
     };
+
+    // quit does not need any of the state from the KeyboardWalk class
+    void quit();
+
```

Comment:
Not needed unless it's called from another file.
Better to avoid adding things to your public interface unless you need to
```suggestion
```

Reply:
If that's the issue, perhaps it should be `private static`?

Reply:
Still would have to be in the header file

Reply:
Right. I am suggesting moving it back into the header, and making it a private, static function declared in the header

Reply:
Personal preference, my preference is to keep the headers with as little in them as possible. Less reasons to trigger a recompilation of files

Reply:
Fair enough. This should be good to rereview then